### DISCUSSION TITLE
Welcome to Faster Discussions!

### DISCUSSION THREAD

**Discussion by Ry8ics** [Ry8ics/Faster#1] [General]
## 👋 Welcome!
Hello, if you entered here you probably have some questions or perhaps some proposals for improving the software. Just let me know, and I will try and help out.

**Comment by thazepl** [comment]
Hi,

I am a Ph.D. student and I just started working with OpenFAST for my thesis. I wanted to ask if Faster is compatible with OpenFAST v.3.3.0, also, I couldn't reach the User documentation. How can I reach it? 

Thank you for sharing Faster,

Best regards,
Engin

**Comment by Ry8icks** [comment]
Sorry for not having responded, I have totally forgotten that I made this repo. Yes, Faster should be compatible with any foreseeable version of Openfast, but you probably already know that by now. The user manual is found under “Releases” where you can find a pdf document that you can download.

**Reply by thazepl** [reply]
That's OK! Thank you for your kind response!
